isthmus/src/main/java/io/substrait/isthmus/SubstraitRelNodeConverter.java

```diff
@@ -214,9 +214,10 @@ public RelNode visit(Set set) throws RuntimeException {
     var builder =
         switch (set.getSetOp()) {
           case MINUS_PRIMARY -> relBuilder.minus(false, numInputs);
-          case MINUS_MULTISET -> relBuilder.minus(true, numInputs);
-          case INTERSECTION_PRIMARY -> relBuilder.intersect(false, numInputs);
-          case INTERSECTION_MULTISET -> relBuilder.intersect(true, numInputs);
+          case MINUS_PRIMARY_ALL, MINUS_MULTISET -> relBuilder.minus(true, numInputs);
+          case INTERSECTION_PRIMARY, INTERSECTION_MULTISET -> relBuilder.intersect(
+              false, numInputs);
+          case INTERSECTION_MULTISET_ALL -> relBuilder.intersect(true, numInputs);
```

Comment:
Good call keeping the old mappings in place for now to allow for easier migration.

Can you add a note indicating that the mappings for `MINUS_MULTISET` and `INTERSECTION_PRIMARY` will be removed will be removed in the future.

Reply:
I put this so others have a reference:
`// MINUS_MULTISET and INTERSECTION_PRIMARY are set to be removed (substrait-io/substrait/pull/708)`